ntc_templates/templates/cisco_ios_traceroute.textfsm

```diff
@@ -41,5 +40,6 @@ Entries
   ^.*?(?:(?:\d+\s+msec|![A-Z]|\*|\?)\s+){8}${RTT_RESPONSE} -> Continue
   ^.*?(?:(?:\d+\s+msec|![A-Z]|\*|\?)\s+){9}${RTT_RESPONSE} -> Continue
   ^.* -> Record
+  ^. -> Error
```

Comment:
As noted in my comment the Error directive doesn't do anything based on the Record line's pattern. (So my suggestion on the VRF in start was helpful, however no Error directive was needed in `Entries`. My mistake.)

```suggestion
```

Reply:
Thank you!